src/core/mod.rs

```diff
@@ -266,9 +268,14 @@ impl<MSG> Drop for EventStream<MSG> {
     }
 }
 
+impl<MSG> Default for EventStream<MSG> {
```

Comment:
I don't really like this clippy lint requiring you to have a `Default` implementation.
Can you just disable it globally, please?